src/Oro/ORM/Query/AST/Platform/Functions/Mysql/Base64Length.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Oro\ORM\Query\AST\Platform\Functions\Mysql;
+
+use Doctrine\ORM\Query\AST\Node;
+use Doctrine\ORM\Query\SqlWalker;
+
+use Oro\ORM\Query\AST\Functions\SimpleFunction;
+use Oro\ORM\Query\AST\Platform\Functions\PlatformFunctionNode;
+
+class Base64Length extends PlatformFunctionNode
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getSql(SqlWalker $sqlWalker)
+    {
+        /** @var Node $expression */
+        $expression = $this->parameters[SimpleFunction::PARAMETER_KEY];
+
+        return 'LENGTH(FROM_BASE64(' . $this->getExpressionValue($expression, $sqlWalker) . '))';
```

Comment:
Thus such functionality should be placed in scope of separate bundle, which defines more strict DB requirements.